test/TestLibrary/ExceptionGrain.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+using System.Threading.Tasks;
+using Orleans;
+
+namespace TestLibrary
```

Comment:
What is this loose file without any project?

Reply:
oops, meant to be deleted, thanks